llmfoundry/eval/datasets/in_context_learning_evaluation.py

```diff
@@ -26,6 +25,7 @@
     get_continuation_span,
     get_fewshot_sample_idxs,
     make_padded_input,
+    stop_sequences_criteria,
```

Comment:
what does this do?

Reply:
moves the import from composer to llmfoundry because the function was moved from composer to llmfoundry